src/main/scala/com/typesafe/sbt/packager/archetypes/JavaAppPackaging.scala

```diff
@@ -1,6 +1,6 @@
 package com.typesafe.sbt.packager.archetypes
 
-import sbt._
+import sbt.{given, _}
```

Comment:
leftover from sbt 2.x migration
```suggestion
import sbt._
```